### DISCUSSION TITLE
Option to disable theme syncing across all clients (Define theme per device)

### DISCUSSION THREAD

**Discussion by balance-wtf** [standardnotes/forum#2630] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

Yes, multiple times, know it by heart

**Is your feature request related to a problem? Please describe.**
Not directly

**Describe the solution you'd like**
I'd like to be able to disable theme syncing across web/desktop clients. For example, I'd like to have the desktop app with the "No Distraction" theme turned on while having a second window open in the browser without the theme. Or have one in dark theme, the other in light theme.

**Describe alternatives you've considered**
Well this is pretty straightforward, don't know if there's a simpler solution.

**Additional context**

**Comment by JaspalSuri** [comment]
Hi @aaantoszek,

I've forwarded your feedback to the team for review. Thanks for the suggestion!
